Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -224,3 +192,29 @@ private func cookieFrom(refTag: RefTag, project: Project) -> HTTPCookie? {
 
   return HTTPCookie(properties: properties)
 }
+
+private func fetchProjectAndLiveStreams(projectOrParam: Either<Project, Param>)
```

Comment:
I also think the call-site being a `switchMap` should take care of it but perhaps @mbrandonw can confirm.